MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```
@@ -27,6 +29,7 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 
 	func display(_ viewModel: FeedLoadingViewModel) {
 		if viewModel.isLoading {
+			self.errorView.hideMessage()
```

Instead of the ViewController deciding when to hide the error message based on loading, the Presenter should be responsible using the `FeedErrorViewModel`.

For example, in the `didStartLoadingFeed()` method, the Presenter can call  `errorMessageView.display(FeedErrorViewModel(errorMessage: .none))` to hide the error.

This way, the ViewController doesn't need to decide when to hide the error. It's easier to manage presentation state transitions when all changes are centralized in the presenter.
```suggestion
```